code/datums/traits/negative.dm

```diff
@@ -612,4 +612,27 @@
 		to_chat(quirk_holder, "<span class='danger'>You forgot you were allergic to [allergy.name]!</span>")
 		H.reagents.add_reagent(/datum/reagent/toxin/histamine, rand(5,10))
 		cooldown = TRUE
-		addtimer(VARSET_CALLBACK(src, cooldown, FALSE), cooldown_time)
\ No newline at end of file
+		addtimer(VARSET_CALLBACK(src, cooldown, FALSE), cooldown_time)
+
+/datum/quirk/kleptomaniac
+	name = "Kleptomaniac"
+	desc = "You have an uncontrollable urge to pick up things you see. Even things that don't belong to you."
+	value = -1
+	mob_trait = TRAIT_KLEPTOMANIAC
+	gain_text = "<span class='danger'>You have an unmistakeable urge to grab nearby objects.</span>"
+	lose_text = "<span class='notice'>You no feel the urge to steal.</span>"
+	medical_record_text = "Patient has an uncontrollable urge to steal."
+
+/datum/quirk/kleptomaniac/on_process()
+	var/mob/living/carbon/H = quirk_holder
+	if(prob(1))
+		var/obj/item/I = locate(/obj/item/) in oview(1, H)
```

Comment:
```suggestion
	if(prob(1))
		var/mob/living/carbon/H = quirk_holder
		var/obj/item/I = locate(/obj/item/) in oview(1, H)
```
You don't need to define the quirk holder on every process since you're not doing anything with it 99% of the time it is defined.